scripts/deploy.sh

```diff
@@ -51,27 +55,45 @@ if [[ "$PUBLIC_URL" =~ ^file:///(.*)/archive.tar.gz ]]; then
 else
     export NIX_ON_DROID_FLAKE_URL="$PUBLIC_URL"
 fi
+export NIX_ON_DROID_BASE_URL="${PUBLIC_URL%/*}"
+
 log "NIX_ON_DROID_CHANNEL_URL=$NIX_ON_DROID_CHANNEL_URL"
 log "NIX_ON_DROID_FLAKE_URL=$NIX_ON_DROID_FLAKE_URL"
+log "NIX_ON_DROID_BASE_URL=$NIX_ON_DROID_BASE_URL"
 
-
-PROOT_HASH_FILE="modules/environment/login/default.nix"
 UPLOADS=()
 for arch in $ARCHES; do
     log "building $arch proot..."
     proot="$(nix build --no-link --print-out-paths ".#prootTermux-${arch}")"
+    proot_hash="$(nix-hash --type sha256 --sri "$proot"/bin/proot-static)"
+    attrs_file="modules/environment/login/proot-attrs/$arch.nix"
+    new_attrs_file="$(mktemp)"
+
+    cat >$new_attrs_file <<EOF
+# WARNING: This file is autogenerated by the deploy script. Any changes will be overridden
+{
+  url = "$NIX_ON_DROID_BASE_URL/bootstrap-$arch.zip";
```

Comment:
this lacks a branch (github:owner/repo/branch case, see above)

Reply:
No, wait, it generally isn't inferrable. The interface will have to be extended to pass the public url of the zip tarball as well =(

Reply:
The use case I want covered is, basically, `someuser` cloning nix-on-droid, creating a branch, changing it and then wanting to test it without adb and with a HTTP server. while we know where to pull the github source tarball from if it's on github (and non-github users assumed to know what they're doing), we don't know what URL will they place the bootstrap tarball at. previously that didn't matter to us, as long as they enter it correctly into the app; now it begins to matter